### ISSUE TITLE
standardize database parameter configuration

### ISSUE THREAD

**Issue by shaycrk** [dssg/triage#647]
Database connection parameters seem to be specified with different keys and connections established in different ways across triage components. We should standardize these to use a single method for connecting to the database. 

A couple of examples (but may not be exhaustive):
- [Triage CLI](https://github.com/dssg/triage/blob/master/src/triage/cli.py#L97)
- [Postmodeling utils](https://github.com/dssg/triage/blob/master/src/triage/component/postmodeling/contrast/utils/aux_funcs.py#L31)
- [Audition Tutorial](https://github.com/dssg/triage/blob/master/src/triage/component/audition/Audition_Tutorial.ipynb)

**Comment by nanounanue** [comment]
We should use the environment variable always, or the connection url

**Comment by jesteria** [comment]
Yeah, my understanding was that we _had_ standardized it, in so far as the Triage library only requires some SQLAlchemy engine, and the Triage CLI accepts either a database URI specified via process environment variable or ... some, _specific_, YAML.

I think this raises inconsistencies which might only remain in tutorials.

_E.g._ I tried to get `postmodeling/contrast/utils/aux_funcs.py:create_pgconn` removed – postmodeling doesn't really use it – but I was told it's used in the iPython notebook tutorial.

So perhaps it's just these tutorials that are at issue?

**Comment by ecsalomon** [comment]
See also the [readme](https://github.com/dssg/triage/tree/master/src/triage/component/results_schema/) and [example database profile](https://github.com/dssg/triage/blob/master/src/triage/component/results_schema/example_db_config.yaml) for the results schema. Despite my improvements, it still contains internal inconsistencies as well as inconsistencies with the triage standard methods.
